serving/docs/lmi/user_guides/release_notes.md

```diff
@@ -0,0 +1,55 @@
+# LMI V10 containers release
+
+This document will contain the latest releases of our LMI containers. For details on any other previous releases, please refer our [github release page](https://github.com/deepjavalibrary/djl-serving/releases)
+
+## Release Notes
+
+### Release date: June 6, 2024
+Check out our latest [Large Model Inference Containers](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#large-model-inference-containers).
+
+### Key Features
+#### LMI container
```

Comment:
You need a newline to separate headings from bullet point lists. Adjacent lines are treated as in the same paragraph in markdown. Technically you should have newlines around every heading, list, code, etc. Some tools parse it anyway, but the mkdocs on our docs site messes up that combination

Reply:
Thanks Zac, Added new lines